repo/packages/M/mr-redis/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://github.com/mesos/mr-redis/blob/master/logo-small.png",
```

Comment:
These need to look like this:

`https://raw.githubusercontent.com/mesos/mr-redis/master/logo-small.png`